DancingGoat/CultureMissingMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DancingGoat
+{
+    public class CultureMissingMiddleware
+    {
+        private readonly string[] _cultures = { "en-US", "es-ES" };
```

Comment:
can we unify the configuration of available languages with the startup class? this is yet another place where we define the culture codes, this becomes unmaintainable